api-server/rockcraft.yaml

```diff
@@ -0,0 +1,116 @@
+name: kfp-api
+summary: An image for using Kubeflow pipelines API
+description: An image for using Kubeflow pipelines API
+version: "2.0.0-alpha.7_22.04_1"
+base: ubuntu:20.04
```

Comment:
Should base be 22.04?

Reply:
Keeping 20.04 for now for some reason I cant install python 3.7 in part on 22.04